modules/system/console/MixInstall.php

```diff
@@ -184,7 +184,8 @@ public function handle(): int
                 $this->info(
                     sprintf('Adding %s (%s) to the workspaces.packages property in package.json', $name, $package['path'])
                 );
-                $workspacesPackages = array_merge($workspacesPackages, [$package['path']]);
+                //Since director seperator differently on Windows platform, each "\" replaced with "/"
+                $workspacesPackages = array_merge($workspacesPackages, [Str::replace("\\","/",$package['path'])]);
```

Comment:
While writing to the package.json, all "\" characters converted to "/" to output a unix based path.